languages/elixir/exercises/concept/secrets/.meta/example.ex

```diff
@@ -23,9 +23,9 @@ defmodule Secrets do
     fn x -> Bitwise.^^^(x, secret) end
   end
 
-  def secret_combine(f, g) do
+  def secret_combine(secret_function1, secret_function2) do
     fn x ->
-      g.(f.(x))
+      secret_function2.(secret_function1.(x))
```

Comment:
Naming functions `f` and `g` is a math convention that not everyone might recognize. I think  we're safer using a more verbose name here.